avaframe/out3Plot/outDistanceTimeAnalysis.py

```diff
@@ -284,9 +292,127 @@ def plotMaskForMTI(cfgRangeTime, bmaskRange, bmaskAvaRadar, bmaskAvaRadarRangesl
+
+        # make or update max velocity line and text
+        maxVel_point = getMaxVelocityPoint(width, height, maxVel, v_diag)
+        if maxVel < v_diag:
+            xLoc = maxVel_point[0] - 0.04
+            yLoc = maxVel_point[1] - 0.004
+        else:
+            xLoc = maxVel_point[0]
+            yLoc = maxVel_point[1] + 0.004
+        if first:
+                event_ax.text(xLoc, yLoc, 'max %.1f' % maxVel, size=5, fontweight='bold',
+                              color='r', transform=ax.transAxes, gid=j, va='center', ha='left',
+                              bbox=dict(facecolor='white', alpha=0.5))
+        else:
+                event_ax.texts[text_element_nr+1].set_position((xLoc, yLoc))
+                l_vmax.set_data([point[0][0], maxVel_point[0]], [point[0][1], maxVel_point[1]])
+    # ---- internal functions end ----
+
+    # keep track of which text elements belong to this legend thing...
+    initital_nr_of_text_elements = len(ax.texts)
+
+    # get legend points in axes coordinates
+    point = getVelocityPoints(1, 1, width, height, 1, 1)
+
+    # add legend title
+    ax.text(point[1][0], point[1][1], 'Approach \n velocity [m/s]', size=textsize,
+        transform=ax.transAxes, va='top', ha='right', fontweight='bold',
+        bbox=dict(facecolor='white', alpha=0.5))
 
+    # plot boundary of velocity box
+    ax.plot([point[0][0], point[1][0]], [point[0][1], point[1][1]], color='k', linewidth=lw, transform=ax.transAxes) # top hor
+    ax.plot([point[1][0], point[5][0]], [point[1][1], point[5][1]], color='k', linewidth=lw, transform=ax.transAxes) # re vert
+    ax.plot([point[5][0], point[9][0]], [point[5][1], point[9][1]], color='k', linewidth=lw, transform=ax.transAxes) # bottom hor
+    ax.plot([point[9][0], point[0][0]], [point[9][1], point[0][1]], color='k', linewidth=lw, transform=ax.transAxes) # le vert
+
+    # these are the diagonal lines referring to velocity
+    for j in range(2, len(point) - 1):
+        x1 = [point[0][0], point[j][0]]
+        y1 = [point[0][1], point[j][1]]
+        ax.plot(x1, y1, color='k', linewidth=0.5, transform=ax.transAxes)
+
+    # draw red line for max velocity
+    xd_intervall = max(ax.get_xlim()) - min(ax.get_xlim())
+    yd_intervall = max(ax.get_ylim()) - min(ax.get_ylim())
+    v_diag = (yd_intervall / xd_intervall)
+    p_vmax = getMaxVelocityPoint(width, height, maxVel, v_diag)
+    l_vmax = ax.plot([point[0][0], p_vmax[0]], [point[0][1], p_vmax[1]], color='r', linewidth=1.0,
+                     transform=ax.transAxes)[0]  # return handle not list
+
+    # make the text, e.g. the mandatory first=True call
+    zoom_lvl_change(ax, first=True)
+
+    # connect callback
+    ax.callbacks.connect('xlim_changed', zoom_lvl_change)
+    ax.callbacks.connect('ylim_changed', zoom_lvl_change)
+
+
+def rangeTimeVelocityLegend_depreciated(timeSteps, rangeGates, ax, maxVel, width=0.25, height=0.25, lw=0.5, textsize=6):
```

Comment:
I assume this is meant to be 'deprecated'? 
If so, then there's no point in keeping it, so just remove these functions